content/docs/hardware/v00.03.12/build-instructions/_index.de.md

```diff
@@ -21,8 +21,9 @@ Das fertig bestückte PCB eines OBS v00.03.
 Diese Anleitung setzt voraus, dass die Grundtechniken wie Löten, das Abisolieren von Kabeln und das Crimpen von JST-Verbindern bereits bekannt sind. Für letzteres empfehlen wir das [JST-Crimp-Tutorial](https://www.youtube.com/watch?v=jHfYzrSF4pY). Warnhinweise und Tipps zu Werkzeugbenutzung finden sich am Anfang der Bauanleitung für das [Vorgängermodell v00.02]({{<relref "/docs/hardware/v00.02/build-instructions">}}).
 
 ## Schritt 1: SMD-Widerstände
+In Revision 00.03.12 können die Widerstände R6 und R7 entweder als SMD- oder als THT-Widerstände ([Through Hole Technology](https://commons.wikimedia.org/wiki/File:3_Resistors.jpg)) gelötet werden, je nachdem welche man zur Verfügung hat. Falls SMD-Widerstände gewählt werden, sollten die jetzt eingelötet werden. THT-Widerstände werden in einem späteren Schritt gelötet.
 
-* Widerstände R6 und R7 anbringen:
+* SMD-Widerstände R6 und R7 anbringen (Fotos zeigen eine ältere Platinen-Revision):
```

Comment:
Die Widerstände sollten jetzt THT sein. Sie haben zwar beide noch den SMD-Footprint zusätzlich (um alte bauteile loszuwerden), aber dabei ist ein Fehler unterlaufen: Wenn R7 SMD oder THT auf der Oberseite platziert wird, passt das eine Sensorboard nicht mehr drauf.

Stattdessen löten wir R6 als THT oder SMD normal , R7 als THT in SMD-Lötweise auf die Unterseite. Klingt kompliziert, ist aber ganz einfach. Wir haben auch schon ein Bild dafür, müssen wir nur noch hochladen :/

Reply:
"THT in SMD-Lötweise":

* Board umdrehen
* Eines der Löcher wie ein SMD-Pad mit Lötzinn versehen
* Widerstand seitlich drauf schieben, dabei Lötzinn (wie bei SMD eben) verflüssigen, Widerstand so fixieren
* Anderes Bein anlöten
* Keines der Beine geht durch das Loch durch.
* Wenig Lötzinn verwenden, sonst sickert das durchs Loch und bildet einen Hubbel auf der anderen (Vorder-) Seite, der ist dann dem Sensorboard im Weg.

Ärgerlicher Bug, aber jetzt sind schon 250 Platinen hergestellt ;)

Reply:
Danke für die Info. Möchtest Du dann eh zeitnah das Foto aktualisieren? Ansonsten kann ich das auch anpassen. Ich warte noch auf Teile, daher habe ich noch nicht angefangen zu löten.

Wegen der THT in SMD-Lötweise: Evtl. ist dann auch was Tesafilm auf der Oberseite hilfreich, damit da kein Lötzinn durchkommt oder man könnte die Beine von der Rückseite durchstecken und anlöten und dann sauber auf der Vorderseite abknipsen.

Reply:
> Tesafilm

Klasse Tip! Danke, das probiere ich direkt aus wenn ich wieder einen Lötkolben in der Hand halte.